format.js

```diff
@@ -65,6 +65,10 @@ function makeEncryptionFormat() {
     return keyring.group.has(recp)
   }
 
+  function isPoBoxId(recp) {
```

Comment:
yeah i knew this was weird/bad but i just copied how isGroupId was used and implemented. maybe better to change both this and that

Reply:
i think it also worked correctly since `has` would only work if it had gone through a check to get added in the first place. anyway, i've changed it to make more sense now